spec/components/input/TextComponent.spec.tsx

```diff
@@ -12,17 +15,23 @@ describe('TextComponent Component', () => {
     });
 
     it('render a TextField with default props and initial value and test if a input html tag is used', () => {
+      const onChange = jest.fn();
       const wrapper = shallow(
-        <TextComponent type={Types.TEXTFIELD} value="Test" />
+        <TextComponent
+          type={Types.TEXTFIELD}
+          value="Test"
+          onChange={onChange}
+        />
       );
       expect(wrapper.length).toEqual(1);
       expect(wrapper.hasClass('tk-input-group')).toBe(true);
       expect(wrapper.find('input.tk-input').length).toBe(1);
       expect(wrapper.find('input.tk-input').prop('value')).toEqual('Test');
     });
     it('render a TextArea with default props and initial value and test if a textarea html tag is used', () => {
+      const onChange = jest.fn();
       const wrapper = shallow(
-        <TextComponent type={Types.TEXTAREA} value="Test" />
+        <TextComponent type={Types.TEXTAREA} value="Test" onChange={onChange} />
```

Comment:
we do we need `onChange={onChange}`

Reply:
Otherwise there is a React warning because we pass a value without a onChange